app/views/foods/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= render 'shared/navigation' %>
+
+<p style="color: green"><%= notice %></p>
+
+<div class="top">
+  <h1>Foods</h1>
+  <p>
+    <%= link_to "Add New food", new_food_path %>
+  </p>
+</div>
+<div class="food_container">
+  <div class="foods">
+    <table class="table table-striped">
+    <thead>
+      <tr>
+        <th>Food</th>
+        <th>Measurement unit</th>
+        <th>Unit price</th>
+        <th>Actions</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @foods.each do |food| %>
+        <% if can?(:manage, food) %>
+          <%= render food %>
+        <% end %>
```

Comment:
- [ ] As requested by the previous reviewer, you should only set @foods to the logged in user. So that here, you don't need to operate and consume time to load all of the foods.

Reply:
well resolved! ✔️